_test/test_sqw_class/test_dnd_copy.m

```diff
@@ -0,0 +1,41 @@
+classdef test_dnd_copy < TestCase
+
+    properties
+        sqw_file_1d_name = 'dnd_2d.sqw';
+        sqw_files_path = '../test_sqw_file/';
+
+        test_dnd_2d_fullpath = '';
+    end
+
+    methods
+
+        function obj = test_dnd_copy(~)
+            obj = obj@TestCase('test_dnd_copy');
+
+            test_dnd_file = java.io.File(pwd(), fullfile(obj.sqw_files_path, obj.sqw_file_1d_name));
+            obj.test_dnd_2d_fullpath = char(test_dnd_file.getCanonicalPath());
+        end
+
+        function test_copy_returns_object_with_identical_data(obj)
+            d2d_obj = d2d(obj.test_dnd_2d_fullpath);
+            d2d_copy = copy(d2d_obj);
+
+            assertEqualToTol(d2d_copy, d2d_obj);
+        end
+
+        function test_copy_returns_distinct_object(obj)
+            d2d_obj = d2d(obj.test_dnd_2d_fullpath);
+            d2d_copy = copy(d2d_obj);
+
+            d2d_copy.s(1:10) = 1e1000;
```

Comment:
Not really important, but `1e1000` evaluates to `inf`. Would it be more explicit to just use `inf` here?